backends/advanced/Docs/README.md

```
@@ -70,12 +70,12 @@ Welcome to chronicle! This guide provides the optimal reading sequence to unders
 
 ## 🔍 **Configuration & Customization**
 
-### 6. **Configuration File** → `../memory_config.yaml`
+### 6. **Configuration File** → `../config.yml`
 **Central configuration for all extraction**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)